tests/test_unsupported.py

```diff
@@ -308,26 +309,14 @@ def test_error_cases(self, case):
 
         settings = _get_all_settings(Mock(indent=4, extension=False), env=env)[lang]
         ext = settings.ext
-        source_data = case
-        expected_error = TEST_ERROR_CASES[case]
-        is_script = has_main(source_data)
-        # TODO: Fix and run the Python snippets
-        # c.f. https://github.com/adsharma/py2many/pull/254/files#r637588902
-        # This if is unnecessary if the lang is constant.
-        if ext in [".dart", ".kt", ".rs"] and not is_script:
-            source_data = f"def main():\n  {source_data}"
-        print(f">{source_data}<")
-        tree = ast.parse(source_data)
-        astpretty.pprint(tree)
-        proc = run([sys.executable, "-c", source_data], capture_output=True)
-        if proc.returncode:
-            raise RuntimeError(f"Invalid case {case}:\n{proc.stdout}{proc.stderr}")
-        try:
-            _transpile([pathlib.Path("stdin.py")], [tree], settings,)[
-                0
-            ][0]
-        except Exception as e:
-            assert type(e) == expected_error
+        source_data = "from ctypes import c_int8 as i8, c_int16 as i16\n" + case
+        tree = get_tree(source_data, ext)
+        # This is supposed to raise a TypeError, however it doesn't.
+        _transpile(
```

Comment:
_transpile() is eating the exception. Perhaps call _transpile_one()?
I tested that the TypeError is being produced when run via py2many.py --rust=1 /tmp/foo.py

Reply:
Will do.

Reply:
done